vcs/Canvas.py

```diff
@@ -4054,6 +4054,8 @@ def __plot(self, arglist, keyargs):
                     dn.g_name = arglist[4]
                     dn.array = arglist[:2]
                     dn.backend = returned_kargs
+                    if "continents" in keyargs:
+                        dn._continents = keyargs["continents"]
```

Comment:
After browsing the source code, I'm not seeing how this value stored on the `display` is ever queried to avoid plotting the continents.  To me it looks like the decision is made inside our `vcs2vtk.genGrid(...)` function, based solely on the type of data we get.  But I guess it's working, so it's fine with me.

Reply:
you're right, it's sort of obscure but i think it's there: https://github.com/CDAT/vcs/blob/master/vcs/VTKPlots.py#L491

Reply:
Ah, you referred to it with an underscore in your fix, so I was searching for it that way, forgetting about how you typically make "properties" out of those attributes.  Thanks.